example/responseWithVariantsAndScoreSummary.json

```diff
@@ -0,0 +1,183 @@
+{
+  "request": {
+    "query": "Blubergurken",
+    "first": 0,
+    "count": 24,
+    "serviceId": "F53ABAB42D7931BE13532AFCA1A95CCE",
+    "usergroup": "foo",
+    "userId": "cd7984ec-e0c5-4bfb-a925-d607668153cd",
+    "order": {
+      "field": "salesfrequency",
+      "relevanceBased": true,
+      "direction": "DESC"
+    }
+  },
+  "result": {
+    "metadata": {
+      "effectiveQuery": "Blubbergurken",
+      "totalResults": 1337,
+      "requestId": "9cd42225-90d0-4858-bcb6-b05f33d8ec5e",
+      "searchConcept": "Seeds",
+      "currencySymbol": "€",
+      "landingpage": {
+        "name": "New arrivals",
+        "url": "https://example.org/new_stuff.html"
+      },
+      "promotion": {
+        "name": "Blubbergurken Brand",
+        "url": "https://example.org/top_brands/Blubbergurken_International_Inc.html",
+        "imageUrl": "https://example.org/top_brands/blubbergurken_international.png"
+      }
+    },
+    "variant": {
+      "name": "sdym",
+      "correctedQuery": "Blubbergurken"
+    },
+    "items": [
+      {
+        "id": "123ab",
+        "url": "https://example.org/product.html",
+        "imageUrl": "https://example.org/product.png",
+        "name": "Blubbergurken Seeds",
+        "highlightedName": "<span class=\"flEmphasis\">Blubbergurken</span> Seeds",
+        "price": 13.37,
+        "ordernumbers": ["0012BLUB-42"],
+        "matchingOrdernumber": "34567",
+        "score": 4.667,
+        "summary": "These are some very nice seeds.",
+        "properties": {
+          "overriddenPrice": "15.00",
+          "taxRate": "20"
+        },
+        "productPlacement": "Seeds spring 2020",
+        "pushRules": [
+          "Seeds",
+          "Cucumbers"
+        ],
+        "attributes": {
+          "cat": [
+            "Gardening"
+          ],
+          "vendor": [
+            "Blubbergurken International Inc."
+          ]
+        },
+        "bestVariant": "123ab-A",
+        "variants": [
+          {
+            "id": "123ab-A",
+            "url": "https://example.org/product-a.html",
+            "imageUrl": "https://example.org/product-a.png",
+            "name": "Blubbergurken Seeds - Class A",
+            "price": 15.00,
+            "ordernumbers": ["0012BLUB-42-A"],
+            "matchingOrdernumber": "",
+            "score": 4.667,
+            "summary": "These are some very nice seeds.",
+            "properties": {
+              "overriddenPrice": "15.00",
+              "taxRate": "20"
+            },
+            "attributes": {
+              "cat": [
+                "Gardening"
+              ],
+              "vendor": [
+                "Blubbergurken International Inc."
+              ]
+            }
+          },
+          {
+            "id": "123ab-A",
```

Comment:
well there are duplicate IDS, so which one will be chosen in this case?

Reply:
That's actually an error in the fixture - duplicate IDs within one shop are not possible.